src/App.jsx

```diff
@@ -1,34 +1,24 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
 import "./App.css";
+import Header from "./components/header/Header";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getArticles } from "./utils/api";
 
 function App() {
-  const [count, setCount] = useState(0);
+  const [items, setItems] = useState([]);
+  const { items_types } = useParams();
+
+  useEffect(() => {
+    getArticles(items_types).then((articlesFromApi) => {
+      setItems(articlesFromApi);
+    });
+  }, [items_types]);
+  console.log("articleApi", items);
 
   return (
-    <>
-      <div>
-        <a href="https://vite.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>nc news hello</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <div>
+      <Header></Header>
```

Comment:
You are only rendering a Header at the moment. The ticket states that you should be rendering all of the articles.